cpp/include/raft/core/detail/nvtx.hpp

```diff
@@ -121,20 +117,12 @@ inline auto hsv2rgb(float h, float s, float v) -> uint32_t
  */
```

Comment:
Could you update the docstring? There is no more lookup.

Reply:
Sure, thanks for noting this!